src/webgpu/format_info.ts

```diff
@@ -1931,6 +1931,21 @@ export function isCompressedTextureFormat(format: GPUTextureFormat) {
   return format in kCompressedTextureFormatInfo;
 }
 
+export function isTextureFormatUsableAsStorageFormat(
+  format: GPUTextureFormat,
+  isCompatibilityMode: boolean
+) {
+  if (isCompatibilityMode) {
+    switch (format) {
+      case 'rg32float':
+      case 'rg32sint':
+      case 'rg32uint':
+        return false;
+    }
+  }
+  return kTextureFormatInfo[format].color?.storage;
```

Comment:
This could return `boolean | undefined` which I think is unwanted. Maybe return `false` when `color` is undefined (depth and stencil seems to always have `storage === false`)

Reply:
Done